{"uuid": "fedb89e8-cac2-477c-bdba-db39de0e2c3f", "counter": 53, "stdout": "\u001b[0;31mfatal: [demo03]: FAILED! => {\"changed\": false, \"msg\": \"One or more supplied key could not be found in the database.\"}\u001b[0m\r\n\u001b[0;36m...ignoring\u001b[0m", "start_line": 57, "end_line": 59, "runner_ident": "100796ab-a203-48a2-9f08-ebbb42174866", "event": "runner_on_failed", "pid": 186390, "created": "2025-05-15T22:40:28.891118+00:00", "parent_uuid": "005056b7-f67d-2a59-52ac-000000000014", "event_data": {"playbook": "/tmp/tmpzonf5ll6.yml", "playbook_uuid": "35daf04b-d1d6-44d6-ae63-58f6f0c050f7", "play": "Instalaci\u00f3n y configuraci\u00f3n de Pure-FTPd para hosts", "play_uuid": "005056b7-f67d-2a59-52ac-000000000002", "play_pattern": "demo03", "task": "Verificar si el usuario FTP existe", "task_uuid": "005056b7-f67d-2a59-52ac-000000000014", "task_action": "getent", "resolved_action": "ansible.builtin.getent", "task_args": "", "task_path": "/tmp/tmpzonf5ll6.yml:146", "host": "demo03", "remote_addr": "demo03", "res": {"msg": "One or more supplied key could not be found in the database.", "invocation": {"module_args": {"database": "passwd", "key": "ftpuser", "fail_key": true, "service": null, "split": null}}, "_ansible_no_log": false, "changed": false}, "start": "2025-05-15T22:40:27.962723+00:00", "end": "2025-05-15T22:40:28.890760+00:00", "duration": 0.928037, "ignore_errors": true, "event_loop": null, "uuid": "fedb89e8-cac2-477c-bdba-db39de0e2c3f"}}